plugins/generator-1.18.2/forge-1.18.2/procedures/entity_pathfind_to.java.ftl

```diff
@@ -0,0 +1,2 @@
+if (${input$entity} instanceof PathfinderMob _entity)
+	_entity.getNavigation().moveTo(${input$x}, ${input$y}, ${input$z}, ${input$speed});
```

Comment:
Use toInt and other optimizers as other procedures do

Reply:
it doesn't require integers

Reply:
If it uses float numbers, there is also a method for that (`opt.toFloat(...)`)